processor/jqprocessor/README.md

```diff
@@ -0,0 +1,32 @@
+# jq processor
+
+Supported pipeline types: logs
+Status: in development
```

Comment:
this can be generated. Try 
```suggestion
<!-- status autogenerated section -->
<!-- end autogenerated section -->
```

then do the following:
```
cd cmd/mdatagen
go install .
cd ../../processor/jqprocessor
mdatagen metadata.yaml
```